### ISSUE TITLE
Multiple cache folders with single ImageLoader instance

### ISSUE THREAD

**Issue by hubcin** [nostra13/Android-Universal-Image-Loader#899]
I want to cache images in different folders and be able to clean those folders separately with just single instance of ImageLoader. Just not being able to find out how.

**Comment by hubcin** [comment]
Yeah, I have to store user profile images in separate folder, I just write a new class that extends the ImageLoader directly, then initiate two Loaders(one for profile image , one for other images) at the start of application. Then I configure the ProfileImageLoader with different disk cache, that is ProfileDiskCache(extends from UnlimitedDiskCache). Try this approach, it will do the work. But which confusing me is that neither of these 2 ImageLoaders seems to caching images into disk storage, I'm saying this because when I'm offline, I will not see those images that have already been downloaded from internet.

**Comment by AizazAZ** [comment]
Well, I have generated keys myself :+1:  and it's working. Ideally should have only one instance of ImageLoader.

**Comment by hubcin** [comment]
yes, I've just got this and already put it into my project, still need some time to figure out how to make the most out of it, thanks for sharing :)
